### DISCUSSION TITLE
Possible to change the home page title as appears in Google Search results?

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#43] [Q&A] [answered]
While I am very pleased with Buttondown, I'd rather its branding not be so prominent in Google Search. Something like "Something Good • Mark Slutsky" would suit me a lot better, or just the name of the newsletter. Is this possible or maybe possibly possible in the future?

<img width="678" alt="image" src="https://github.com/user-attachments/assets/46a59515-eddf-4b70-a2b4-f0f0e41a5e6f">

**Comment by jmduke** [comment, marked as answer]
It is — if you cough up the $79/mo and enable Whitelabeling 😅.

**Reply by krangor** [reply]
![](https://media1.tenor.com/m/MnDVObkGxh8AAAAC/the-simpsons-nevermind.gif)
